CONTRIBUTING.md

```diff
@@ -180,4 +180,21 @@ make misspell-correction
   - Send staging tag as PR for review.
 - Create a tag `v{version}` on the merged PR and push remote.
 
+## Merging existing ECS conventions
+
+The Elastic Common Schema (ECS) is being merged into OpenTelemetry Semantic
+Conventions per [OTEP 222][otep222]. When adding a semantic convention that
+exists in some form in ECS, consider the following guidelines:
+
+- Prefer using the existing ECS name when possible. In particular:
```

Comment:
Recently it was introduced container attributes from [OCI](https://github.com/open-telemetry/semantic-conventions/pull/159). What will be our guidelines when:

- Another standard (such OCI in this case) has attributes different from the ones from ECS? Will we still take ECS? Will ECS adapt to the other, potentially OSS standard?

Reply:
I think that in https://github.com/open-telemetry/semantic-conventions/pull/159 we tried to follow the OCI spec and not directly use any naming convention. In this there was no breaking-change trade off, right?

Reply:
Yes, I just mentioned that as an example of another open standard that could clash with ECS.

Reply:
@joaopgrassi I think that would be rather an edge case that we would discuss case by case anyways, WDYT?

Reply:
I would take this as 'comparable evidence' on the rule for 'proposing a name that differs from the ECS convention'. Maybe we can explicitly add "prior work on a different standard" as a piece of evidence that can be 'used as evidence'?

Reply:
In my understanding this case is well covered by this point:

> * If proposing a name that differs from the ECS convention, provide usage data, user issue reports or feature requests or comparable evidence about the alternatives.

If there's a "competing" standard, I think it needs to be evaluated on a case-by-case basis whether the additional benefits for users of being consistent with the "competing" standard outweigh the pain of breaking ECS.

Reply:
Added this explicitly on that bullet point for clarity